homeassistant/components/group/switch.py

```diff
@@ -168,6 +168,6 @@ def async_update_group_state(self) -> None:
             self._attr_is_on = None
         else:
             # Set as ON if any / all member is ON
-            self._attr_is_on = self.mode(list(map(lambda x: x == STATE_ON, states)))
+            self._attr_is_on = self.mode(state == STATE_ON for state in states)
 
         self._attr_available = any(state != STATE_UNAVAILABLE for state in states)
```

Comment:
This isn't technically a part of this PR, but this should may be easier to read (and marginally more performant).
```suggestion
        self._attr_available = STATE_UNAVAILABLE not in states
```

Reply:
Nevermind - you only want to mark unavailable if all are so.